CMakeLists.txt

```diff
@@ -69,7 +69,7 @@ if(MSVC)
 else()
   option(BUILD_SHARED_LIBS "Build shared libraries" ON)
 endif()
-option(KIDO_BUILD_OSGKIDO "Build osgKido library" ON)
+option(KIDO_BUILD_OSGKIDO "Build osgKido library" ON) # TODO(JS): we probably should remove this option or add another build options for other extensions as well for consistency.
```

Comment:
Okay, as KIDO 0.1.0 should be released as soon as possible, I'll add them for KIDO 1.0.0.